src/components/About/About.tsx

```diff
@@ -0,0 +1,51 @@
+import { Anchor, Container, Grid, Group, Image, Modal, Text, Title } from '@mantine/core';
+
+import { useAppSelector } from '@/redux/hooks';
+
+interface Props {
+  opened: boolean;
+  close: () => void;
+}
+export function About({ opened, close }: Props) {
+  const openSpaceVersion = useAppSelector((state) => state.version.openSpaceVersion);
+
+  function osVersionNumber(): string {
+    if (!openSpaceVersion) {
+      return 'Fetching OpenSpace version...';
+    }
+
+    function formatVersion(version: {
+      major: number;
+      minor: number;
+      patch: number;
+    }): string {
+      return version.major !== 255 && version.minor !== 255 && version.patch !== 255
+        ? `${version.major}.${version.minor}.${version.patch}`
+        : 'Custom';
+    }
+
+    return `OpenSpace version: ${formatVersion(openSpaceVersion)}`;
+  }
+  return (
+    <Modal opened={opened} onClose={close} title={'About OpenSpace'} size={'40%'}>
+      <Grid>
+        <Grid.Col span={4}>
+          <Image src={'openspace-logo.png'} alt={'OpenSpace logo'} w={'100%'} />
+        </Grid.Col>
+        <Grid.Col span={8}>
+          <Title order={1}>OpenSpace</Title>
```

Comment:
Hoa about using a `Stack` around this column content?  To give some natural spacing :)